Makefile

```diff
@@ -14,7 +14,6 @@
 
 .-PHONY: cluster deploy deploy-continuous logs checkstyle check-env
 
-ZONE=us-west1-a
```

Comment:
is there a reason for removing this?

Reply:
Yes, but actually no.

Accidentally included it in the commit